liwords-ui/src/App.tsx

```diff
@@ -204,6 +218,24 @@ const App = React.memo(() => {
     }
   }, [getFriends, pendingFriendsRefresh]);
 
+  useEffect(() => {
+    const notifyInfo = getTermsNotify();
+    if (loggedIn && !notifyInfo?.notified) {
+      notification.info({
+        message: `Our terms of service were updated on ${notifyInfo.changed}. Click to read them.`,
+        duration: 0,
```

Comment:
What do we think about making this fade away eventually, after 5-10 seconds? I worry that a persistent notification might distract people when they join a game?

Reply:
We should ask Conrad. More like 20 seems better to me for this. It could also just close if they start a game or whenever we choose.
Tangentially, are you still not seeing the X to close those notifications? Wonder if you have user CSS on from the community styles or something.

Reply:
if we're not forcing people to accept anything, why don't we just send an email to all users instead of an in-app notification?

That seems less distracting, and the pattern companies usually use AFAIK.